modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/wstep/impl/CertificateEnrollmentServiceImpl.java

```diff
@@ -342,6 +342,10 @@ public String prepareWapProvisioningXML(
             String msg = "Error occurred in while getting wap-provisioning.xml file.";
             log.error(msg, e);
             throw new WAPProvisioningException(msg, e);
+        } catch (SyncmlMessageFormatException e) {
+            String msg = "Error occurred in while getting CA and Root certificates.";
```

Comment:
Error in grammar - should be "Error occurred while getting CA and Root certificates."